layouts/shortcodes/home-section-resources.html

```diff
@@ -0,0 +1,75 @@
+<!-- 
+  Copyright (c) 2019 Eclipse Foundation, Inc.
+
+  This program and the accompanying materials are made available under the
+  terms of the Eclipse Public License v. 2.0 which is available at
+  http://www.eclipse.org/legal/epl-2.0.
+
+  Contributors:
+    Yi Liu <yi.liu@eclipse-foundation.org>
+
+  SPDX-License-Identifier: EPL-2.0
+-->
+
+<section class="container">
+  <h2>Resources</h2>
+  <div class="row">
+    <div class="col-md-6 col-sm-12">
+      <div class="flex-column text-center match-height-item background-white padding-20 margin-bottom-30">
+        <div class="flex-grow">
+          <p><img class="img img-responsive margin-auto padding-20" style="height: 150px; width: 150px;" src="images/resources/whitepaper-icon.svg" alt="white paper icon"></p>
+          <h3>Whitepaper</h3>
+          <p>Learn how open source is fueling digital transformation in Europe and beyond.</p>
+        </div>
+        <p><a class="btn btn-secondary" href="https://outreach.eclipse.foundation/enabling-digital-transformation-white-paper" aria-label="Download enabling digital transformation white paper">Download</a></p>
+      </div>
+    </div>
+    <div class="col-md-6 col-sm-12">
+      <div class="flex-column text-center match-height-item background-white padding-20 margin-bottom-30">
+        <div class="flex-grow">
+          <p><img class="img img-responsive margin-auto padding-20" style="height: 150px; width: 150px;" src="images/resources/factsheet-icon.svg" alt="fact sheet icon"></p>
+          <h3>Fact Sheet</h3>
+          <p>Get to know the Eclipse Foundation and learn about our strategic focus areas.</p>
+        </div>
+        <p><a class="btn btn-secondary" href="https://www.eclipse.org/europe/eclipse-fact-sheet.pdf" aria-label="Download eclipse fact sheet pdf">Download</a></p>
```

Comment:
Anywhere we use the Eclipse brand it should be capitalized. It's our brand, and a proper noun. To not capitalize dilutes/confuses the usage.